### ISSUE TITLE
Label docstrings with versionadded / versionchanged

### ISSUE THREAD

**Issue by reshamas** [data-umbrella/2020-sklearn-sprint#1]
https://github.com/scikit-learn/scikit-learn/issues/15426
